tools/crm_resource.c

```diff
@@ -41,7 +41,6 @@ enum rsc_command {
     cmd_cleanup,
     cmd_clear,
     cmd_colocations,
-    cmd_cts,
```

Comment:
This is for internal use (by python/pacemaker/_cts/audits.py)

Reply:
Where is the `--list-cts` option used in `audits.py`?

Reply:
Thank you, I forgot to check for short options. Which are much harder to search for anyway, since they're not specific on their own and they might not occur on the same line as crm_resource.